Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LocalSettingsSamplePage.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<Page x:Class="Uno.Gallery.Views.Samples.LocalSettingsSamplePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:Uno.Gallery"
+	  xmlns:samples="using:Uno.Gallery.Views.Samples"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:smtx="using:ShowMeTheXAML"
+	  xmlns:android="http://uno.ui/android"
+	  xmlns:ios="http://uno.ui/ios"
+	  xmlns:macos="http://uno.ui/macos"
+	  xmlns:skia="http://uno.ui/skia"
+	  xmlns:wasm="http://uno.ui/wasm"
+	  xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  mc:Ignorable="d android ios macos skia wasm"
+	  x:Name="page">
+
+	<Page.Resources>
+		<ResourceDictionary>
+            <samples:ClipboardSamplePageViewModel x:Key="LocalSettingsSamplePageViewModel"/>
+		</ResourceDictionary>
+	</Page.Resources>
```

Comment:
Is this required? Looks to be a view model for the Clipboard sample page

Reply:
Yeah, I copied the page from the Clipboard sample. I thought about providing a way to provide settings by the user but switched to the fixed key.
I removed the ViewModel.